test/core.js

```diff
@@ -24,10 +24,11 @@ describe('Core methods', function() {
     });
 
     it('should copy simple fields from config', function() {
-      var config = {lint: false, verbose: true};
+      var config = {lint: false, verbose: true, syntax: {}};
       comb.configure(config);
       assert.equal(comb.lint, config.lint);
       assert.equal(comb.verbose, config.verbose);
+      assert.equal(comb.syntaxMap, config.syntax);
```

Comment:
You can't be sure that map was copied because `{}` is the default value it gets even if configuration fails.
I'd offer replacing on line 27 `syntax: {}` with `syntax: {'.css': 'scss'}`, for example.